src/Stratis.Bitcoin/Consensus/ConsensusManager.cs

```diff
@@ -250,7 +253,7 @@ public async Task<ChainedHeaderBlock> BlockMinedAsync(Block block)
 
                     if (fullValidationRequired)
                     {
-                        ConnectBlocksResult fullValidationResult = await this.FullyValidateLockedAsync(partialValidationResult.ChainedHeaderBlock).ConfigureAwait(false);
+                        ConnectBlocksResult fullValidationResult = await this.FullyValidateLockedAsync(new ChainedHeaderBlock(validationResult.BlockToValidate, validationResult.ChainedHeaderToValidate)).ConfigureAwait(false);
```

Comment:
I thought you said that these allocations were removed– We can totally remove this one I guess

Reply:
fixed